extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -491,6 +491,20 @@
 		_t.buttonDown = _t.parentNode.querySelector(o.rollerblind.down);
 		_t.buttonStop = _t.parentNode.querySelector(o.rollerblind.stop);
 
+		_t.hasValue = _t.parentNode.getAttribute("data-has-value") === "true";
+		_t.valueNode = _t.parentNode.parentNode.querySelector(o.formValue);
+		_t.setValuePrivate = function(value) {
+			if (!_t.hasValue) {
+				return;
+			}
+			if (value === "DOWN") {
```

Comment:
> or is there any other use case?

Yes, it is always active and pro-actively assumes a state change when a command is seen on the event bus.
Bindings should be able to deactivate this (this was possible in OH1, but is an open issue for ESH, see https://github.com/eclipse/smarthome/issues/595)